github_copilot_seats/seats.py

```diff
@@ -0,0 +1,113 @@
+import json
+import os
+from datetime import datetime, timezone, timedelta
+from typing import Any
+import autokitteh
+from autokitteh.github import github_client
+from autokitteh.slack import slack_client
+from helpers import github_username_to_slack_user_id
```

Comment:
Order the imports, and split into 3 blocks: standard library, 3rd-party (including AK SDK), and project-internal.

```suggestion
from datetime import datetime, timezone, timedelta
import json
import os
from typing import Any

import autokitteh
from autokitteh.github import github_client
from autokitteh.slack import slack_client

from helpers import github_username_to_slack_user_id
```